spec/setup_spec.lua

```diff
@@ -29,4 +29,77 @@ describe("org-roam.setup", function()
         assert.are.equal(db_path, roam.database:path())
         assert.are.equal(directory, roam.database:files_path())
     end)
+
+    local has_file_like = function(state, arguments)
+        local file_paths = arguments[1]
+        local patterns = arguments[2]
+        for _, pattern in ipairs(patterns) do
+            pattern = string.gsub(pattern, "-", "%%-") .. "$"
+            local match_found = false
+            for _, file_path in ipairs(file_paths) do
+                if file_path:match(pattern) then
+                    match_found = true
+                    break
+                end
+            end
+            if not match_found then
+                print(pattern .. "not found")
+                return false
+            end
+        end
+        return true
+    end
+
+    local say = require("say")
```

Comment:
My own naiveness here. Where does this library come from? Is this something that plenary.nvim provides? Or does it come in from somewhere else? And I don't actually know what say does. Can you explain, maybe adding a comment section above, what this is doing so it's documented for future test contributions?

Reply:
I actually also wondered, when I saw the example code from luassert's `register` function (luassert is the asserting library deployed with busted). I then tried it out and it just works. Turns out it is a hashing library and key-value store used by busted and luassert and maintained - also like busted and luassert - in the lunarmodules project, see [here](https://github.com/lunarmodules/say).

Reply:
I added a comment as suggested by you.